guide/SUMMARY.md

```diff
@@ -0,0 +1,37 @@
+## TELUS Design System
+
+* [Home](README.md)
+* [Getting Started](/getting-started/getting-started.md)
```

Comment:
Does the SUMMARY need to be absolute? Because the SUMMARY is always visible on every page? Make sure to check that relative links work properly from within the SUMMARY.

Reply:
relative links in SUMMARY still work within subpages.